src/services/token.service.js

```diff
@@ -0,0 +1,39 @@
+const { Token } = require('../models/Token.js');
+
+async function save(userId, refreshToken) {
+  const token = await Token.findOne({
+    where: { userId },
+  });
+
+  if (token) {
+    token.refreshToken = refreshToken;
+
+    await token.save();
+
+    return true;
+  }
+
+  await Token.create({ userId, refreshToken });
+
+  return false;
```

Comment:
Consider documenting the return value behavior of the `save` function. It returns `true` if an existing token is updated and `false` if a new token is created. This behavior might be non-intuitive and should be clearly communicated to users of the function.